generateEntityFromLeesStore.py

```diff
@@ -73,13 +73,13 @@
 
 ############  USER INPUT ###########
 # the root directory of your app that contains the angular and aspnet-core folders
-target_dir = "C:\\MyProjects\\Sample\\5.6.0\\"
+target_dir = "..\\"
```

Comment:
As mentioned in the summary I think relative references will work better.  I assumed this script was in the /angular folder, but ideally I think this should assume it's in a `/scripts` folder off the root